admin_guide/pruning_resources.adoc

```diff
@@ -235,6 +235,16 @@ case it fails (the registry cannot be resolved or reached), an alternative
 route that works needs to be provided using this flag. The registry host name
 may be prefixed by `https://` or `http://` which will enforce particular
 connection protocol.
+
+.^|`--prune-registry`
+|In conjunction with the conditions stipulated by the other options, this option controls
+the operation only considers the {product-title} Image API Objects, or also processes
+the corresponding data in the registry.  By default, image pruning processes both the
+Image API Objects and corresponding data in the registry.
+This option makes most sense when there is either a large amount of data in etcd
+and/or a large number of images or blobs. After running with this option set to `false`, you can
```

Comment:
I would phrase it as "This options is useful when you are only concerned with removing etcd content, possibly to reduce the number of image objects, but are not concerned with cleaning up registry storage, or intend to do that separately via hard prune, possibly during an appropriate maintenance window for the registry."